doc/src/manual/getting-started.md

```diff
@@ -3,6 +3,8 @@
 Julia installation is straightforward, whether using precompiled binaries or compiling from source.
 Download and install Julia by following the instructions at [https://julialang.org/downloads/](https://julialang.org/downloads/).
 
+If you are coming to Julia from one of the following languages, then you should start by reading the section on noteworthy differences from [Matlab](@ref Noteworthy differences from MATLAB), [R](@ref Noteworthy differences from R), [Python](@ref Noteworthy differences from Python), [C/C++](@ref Noteworthy differences from C/C++) or [Common Lisp](@ref Noteworthy differences from Common Lisp). This will help you avoid some common pitfalls since Julia differs from those languages in many subtle ways.
```

Comment:
These references don't look right.

Reply:
Easiest way to figure out the link is to just go to the page and click on the headers. This will update the url in your browser. So https://docs.julialang.org/en/v1/manual/noteworthy-differences/#Noteworthy-differences-from-MATLAB-1

should be `(@ref Noteworthy-differences-from-MATLAB-1)`. I think...

Reply:
Thanks! Will fix.

Reply:
I am not sure about the 1. Probably check a few other existing references to see how to map the URLs. The doctests will tell you if you got it right. But if you ahve a source build, just do `make` in docs. It's quite easy and reasonably quick.

Reply:
Without the -1, the docs build and the links seem to work correctly.